@adenzler-nvidia — I registered `basic.example_basic_conveyor` locally and tried turning on the existing output-checking by flipping `check_output=False` → `True` in `add_example_test` (the hardcoded value at [test_examples.py:194](https://github.com/newton-physics/newton/blob/main/newton/tests/test_examples.py#L194), which currently disables it for every example).

The test fails, but not on the inertia warning — it fails on the OpenUSD banner that the test runner emits when [unittest_parallel.py:26](https://github.com/newton-physics/newton/blob/main/newton/tests/thirdparty/unittest_parallel.py#L26) sets `PXR_WORK_THREAD_LIMIT=1`:

```
AssertionError: Unexpected output:
'##################################################################
#  PXR_WORK_THREAD_LIMIT is overridden to '1'.  Default is '0'.  #
##################################################################'
```

The subprocess inherits that env var, libusd re-emits the 3-line banner on stderr, the example-test runner does `print(result.stderr)` ([test_examples.py:176-177](https://github.com/newton-physics/newton/blob/main/newton/tests/test_examples.py#L176-L177)), and `CheckOutput`'s only filter is for `"Module xxx load on device"` lines ([unittest_utils.py:216-218](https://github.com/newton-physics/newton/blob/main/newton/tests/unittest_utils.py#L216-L218)), so the banner trips it. Any other example test would hit the same thing if we turned this on.

Options I can think of:

1. Extend `CheckOutput`'s filter to also skip the PXR banner block.
2. Pop `PXR_WORK_THREAD_LIMIT` from the subprocess env in `add_example_test` so the banner never fires (works fine for examples that don't load USD; could re-introduce the documented OpenUSD thread-safety crash for USD-loading examples).
3. Some other mechanism I'm missing.

Do you have a preferred direction? Happy to take this further but want to confirm the approach before changing shared test infra.

--- a/newton/tests/test_examples.py
+++ b/newton/tests/test_examples.py
@@ -251,6 +251,14 @@ add_example_test(
     test_options={"num-frames": 150},
 )
 
+add_example_test(
+    TestBasicExamples,
+    name="basic.example_basic_conveyor",
+    devices=test_devices,
+    use_viewer=True,
+    test_options={"num-frames": 100},
+)
+
 
 class TestCableExamples(unittest.TestCase):
     pass

